src/hooks/series/useSeriesId.ts

```diff
@@ -0,0 +1,26 @@
+import { useQuery } from 'react-query';
+
+import type { PlaylistItem } from '#types/playlist';
+import { getSeriesByMediaIds } from '#src/services/api.service';
+import { getLegacySeriesPlaylistIdFromEpisodeTags } from '#src/utils/media';
+
+const useGetSeriesId = (item: PlaylistItem) => {
```

Comment:
I would suggest you keep this limited to the query logic so it can be reused everywhere you're calling `getSeriesByMediaIds` 
```suggestion
const useSeriesLookup = (item: PlaylistItem) => {
```